.eslintignore

```diff
@@ -1 +1,2 @@
 shared/js/content-scope/sjcl.js
+unit-test/background/reference-tests/
```

Comment:
ditto - reference tests have js files (with SEMICOLONS!) and we don't want them linted here

Reply:
Is this the only issue? We could could add an exception to this rule in the reference tests repo perhaps? (eslint checks for .eslintrc's in all dirs)

Reply:
there are couple of issues and we don't eslint that script in the reference-tests repo (because it's one 50 line script - https://github.com/duckduckgo/privacy-reference-tests/blob/main/_scripts/linting.js and it didn't seem worth the effort)